packages/flutter/lib/src/painting/border_radius.dart

```diff
@@ -6,152 +6,842 @@ import 'package:flutter/foundation.dart';
 
 import 'basic_types.dart';
 
+/// Base class for [BorderRadius] that allows for text-direction aware resolution.
+///
+/// A property or argument of this type accepts classes created either with [new
+/// BorderRadius.only] and its variants, or [new BorderRadiusDirectional.only]
```

Comment:
Does this line break mess up the docs?  It messes up GitHub's colorization.

Reply:
Pretty sure dartdoc handles this fine, but I'll try to remember to check.

Reply:
yup, looks fine